templates/voting-stats.html

```diff
@@ -0,0 +1,391 @@
+<style>
```

Comment:
Is putting this file directly in `templates/` a good idea? I think moving it to either under `templates/stats/` or `templates/problem/` is better.